ckan/tests/logic/action/test_get.py

```diff
@@ -1741,6 +1741,30 @@ def test_resource_view_show_id_not_found(self):
             logic.NotFound,
             helpers.call_action, 'resource_view_show', id='does_not_exist')
 
+class ShowResourceFileMetadata(object):
```

Comment:
Can there be tests for successful get, and for a resource that isn't a upload?

Reply:
seems all CKAN tests i've found do not write to disk, i'll see if there is other locations i can checkout and reuse.

Reply:
have created a test and was correct, it uses a dummy filesystem and won't give out correct hashs